src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpResponseStream.cs

```diff
@@ -103,7 +103,7 @@ public override Task<int> ReadAsync(byte[] buffer, int offset, int count, Cancel
 
             if (count > buffer.Length - offset)
             {
-                throw new ArgumentException(nameof(buffer));
+                throw new ArgumentException(SR.Argument_BufferNotLongEnough, nameof(buffer));
```

Comment:
Changed.  Would it be appropriate to move to the derived `ArgumentOutOfRangeException` here?  That would likely still require a custom message, given we'd need to mention both `count` and `offset` in such a case.